### PR TITLE
ZCS-9101 Initial changes to replace Flash-chart with ChartJS.

### PR DESCRIPTION
**Pull request by BhavinBathani**

### CODE REVIEW

File: WebRoot/public/admin.jsp [hunk 1]
```diff
@@ -196,7 +196,8 @@
 <script type="text/javascript" src="${contextPath}/yui/2.7.0/element/element-min.js"></script> 
 <script type="text/javascript" src="${contextPath}/yui/2.7.0/datasource/datasource-min.js"></script> 
 <script type="text/javascript" src="${contextPath}/yui/2.7.0/json/json-min.js"></script> 
-<script type="text/javascript" src="${contextPath}/yui/2.7.0/charts/charts-min.js"></script> 
+<script type="text/javascript" src="${contextPath}/yui/2.7.0/charts/charts-min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.js"></script>
```
Comment (BhavinBathani) [inline WebRoot/public/admin.jsp:200]: @silentsakky I need your help here. What is the proper way to include third-party libs in admin-console environment?

**Comment by silentsakky** [comment]
remove .DS-Store files from PR

File: WebRoot/js/zimbraAdmin/statistics/view/ZaGlobalMessageCountPage.js [hunk 2]
```diff
@@ -82,19 +82,19 @@ function () {
 	html[idx++] = "<div class='StatsDiv'>";	
 	html[idx++] = "<div class='StatsImageTitle'>" + AjxStringUtil.htmlEncode(ZaMsg.NAD_StatsHour) + "</div>";	
 	html[idx++] = "<div class='StatsImage'>";
-	html[idx++] = "<div id='loggerchartglobal-message-count-48hours'></div>";	
+	html[idx++] = "<canvas id='loggercanvasglobal-message-count-48hours'></canvas><div id='loggerchartglobal-message-count-48hours'></div>";
```
Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/statistics/view/ZaGlobalMessageCountPage.js:85]: do we need ids for div tag?

File: WebRoot/public/admin.jsp (hunk 1 above)
Comment (BhavinBathani) [reply WebRoot/public/admin.jsp]: Fixed.

**Comment by BhavinBathani** [comment]
> remove .DS-Store files from PR

Fixed.

**Review by silentsakky — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Review by lavegupta — APPROVED** [review]

File: WebRoot/js/zimbraAdmin/statistics/view/ZaGlobalAdvancedStatsPage.js [hunk 3]
```diff
@@ -404,7 +404,12 @@ ZaGlobalAdvancedStatsPage.plotChart = function (id, fields, colDef, newData) {
                 position: 'bottom'
             },
             tooltips: {
-                mode: 'point'
+                mode: 'point',
+                callbacks: {
+                    label: function (tooltipItems, data) {
+                        return data.datasets[tooltipItems.datasetIndex].label + " : " + ZaGlobalAdvancedStatsPage.formatLabel(tooltipItems.value)
```
Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/statistics/view/ZaGlobalAdvancedStatsPage.js:410]: semicolon at end of line

Comment (BhavinBathani) [reply WebRoot/js/zimbraAdmin/statistics/view/ZaGlobalAdvancedStatsPage.js]: Fixed.

**Review by silentsakky — APPROVED** [review]

**Pull request merged**